File: Lokalano-partnerstvo/client/src/app/shared/models/kurs.ts
export interface Kurs {
  id: number;
  naziv: string;
  aktivan: boolean;
  brojPolaznika: number;
  cijena: number;
  datumObjave: Date;
  datumPocetka: Date;
  kursKategorija: string;
  kursKategorijaId: number;
  objavio: string;
  opis: string;
  kratakOpis: string;
  trajanje: number;
  imageUrl: string;
  photo: Photo;
}
export interface Photo {
  id: number;
  imageUrl: string;
  fileName: string;
}
export interface KursToCreate {
  naziv: string;
  aktivna: boolean;
  brojPolaznika: number;
  cijena: number;
  datumObjave: string;
  datumPocetka: string;
  fokus: boolean;
  kursKategorijaId: number;
  opis: string;
  kratakOpis: string;
  trajanje: number;
  imageUrl: string;
}

export class KursFormValues implements KursToCreate {
  naziv = '';
  aktivna = true;
  brojPolaznika = 0;
  cijena = 0;
  datumObjave = Date.now.toString();
  datumPocetka = Date.now.toString();
  fokus = false;
  kursKategorijaId: number;
  opis = '';
  kratakOpis = '';
  trajanje = 0;
  imageUrl = '';

  constructor(init?: KursFormValues) {
    Object.assign(this, init);
  }

}

File: Lokalano-partnerstvo/Core/Entities/Kurs.cs
using System;

namespace Core.Entities
{
    public class Kurs : BaseEntity
    {
        public string Naziv { get; set; }
        public bool Aktivan { get; set; }
        public int BrojPolaznika { get; set; }
        public double Cijena { get; set; }
        public DateTime DatumObjave { get; set; } = DateTime.Now;
        public DateTime DatumPocetka { get; set; } = DateTime.Now;
        public int KursKategorijaId { get; set; }
        public KursKategorija KursKategorija { get; set; }
        public string Objavio { get; set; }
        public string Opis { get; set; }
        public string KratakOpis { get; set; }
        public int Trajanje { get; set; }
        public string  ImageUrl { get; set; }
        public Photo Photo { get; set; }
        public int? PhotoId { get; set; }

        public void AddPhoto(string imageUrl, string fileName)
        {
            var photo = new Photo
            {
                FileName = fileName,
                PictureUrl = imageUrl
            };

            Photo = photo;
        }
        public void RemovePhoto()
        {
            if (Photo != null)
            {
                Photo = null;
            }
        }
    }
}